### PR TITLE
#74 Rendered Hall-1 and Hall-2

### PR DESCRIPTION
**Pull request by paulMoose**
+ cleaned up mock floor plans 7, 9, 10

# Hall-1
image
# Hall-2
image

### CODE REVIEW

**Comment by a-paggabao** [comment]
Can the color of the room names be changed using css?

**Comment by paulMoose** [comment]
> Can the color of the room names be changed using css?

yes

**Comment by jayohanna** [comment]
It seems like Hall-2 is missing the stairs icons on the stairs that lead to Hall-1

File: SD-K9/src/assets/hall/2.svg [hunk 1]
```diff
@@ -0,0 +1,1499 @@
...
+	<line style="fill:none;stroke:#B79065;" x1="224.2" y1="416.3" x2="224.2" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="219.6" y1="416.3" x2="219.6" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="215" y1="416.3" x2="215" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="210.3" y1="416.3" x2="210.3" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="205.7" y1="416.3" x2="205.7" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="201.1" y1="416.3" x2="201.1" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="196.5" y1="416.3" x2="196.5" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="191.8" y1="416.3" x2="191.8" y2="495.5"/>
+	<line style="fill:none;stroke:#B79065;" x1="187.2" y1="416.3" x2="187.2" y2="495.5"/>
+	<g id="stairs-1">
```
Comment (jayohanna) [inline SD-K9/src/assets/hall/2.svg]: Make sure the stairs IDs remain the same between floors!

**Pull request merged**